From 021543b02b6fdb78c9217757736eabac90a647eb Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Sat, 26 Sep 2020 12:37:38 +0200
Subject: [PATCH] port.h: include <sys/time.h>

Include <sys/time.h> to avoid the following build failure on musl:

port.h:361:29: warning: 'struct timeval' declared inside parameter list will not be visible outside of this definition or declaration
     const char *str, struct timeval *tv,
                             ^~~~~~~
portconfig.c: In function 'myconfig':
portconfig.c:586:9: error: variable 'tv' has initializer but incomplete type
  struct timeval tv = { 0, 0 };
         ^~~~~~~

Fixes:
 - http://autobuild.buildroot.org/results/4c0b238186cb2fb2d81807ce006945594f92b2cd

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: https://github.com/cminyard/ser2net/pull/34]
---
 port.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/port.h b/port.h
index 048c5bd..86c3b1d 100644
--- a/port.h
+++ b/port.h
@@ -9,6 +9,7 @@
 #define PORT
 
 #include <netdb.h>
+#include <sys/time.h>
 #include "gbuf.h"
 #include "absout.h"
 
-- 
2.28.0

